examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -101,6 +101,13 @@
 # and a title
 display.title('"tmap_filename" on glass brain without threshold')
 
```

Comment:
I would have liked an extra separation here, so that we have only one figure per cell. It makes things easier to understand for the user.